src/components/TuneLabelInput.vue

```diff
@@ -7,8 +7,13 @@ defineProps<{
 </script>
 
 <template>
-  <div class="tune-label mb-[2px] flex items-center gap-1">
-    <slot />
-    <TuneIconHint :hint="hint" />
+  <div class="tune-label-container flex flex-col">
+    <div class="tune-label mb-[2px] flex items-center gap-1">
+      <slot />
+      <TuneIconHint :hint="hint" />
+    </div>
+    <div v-if="$slots.sublabel" class="tune-sublabel text-sm text-skin-text opacity-60">
+      <slot name="sublabel" />
```

Comment:
Maybe instead of adding sublabel we should reuse hint and add a prop `showSublabel: boolean` to show the hint a sublabel?

Reply:
It will make the interface of the component more intricate. Without reading the documentation/code user will not understand that he needs to set a flag for the purpose of turning the hint into a sublabel. So having a different essence for this looks more straightforward. Additionally, it allows having a sublabel and a hint altogether.

Reply:
Good point